gensim/models/deprecated/old_saveload.py

```diff
@@ -367,7 +367,7 @@ def unpickle(fname):
         Python object loaded from `fname`.
 
     """
-    with smart_open(fname, 'rb') as f:
+    with open(fname, 'rb') as f:
```

Comment:
smart_open **does** have readline now, so the below comment is inaccurate.

Looks like the only reason to read the whole thing into memory is that replace call.

Reply:
removed this comment